$NetBSD: patch-meson.build,v 1.1 2022/11/21 23:55:23 gdt Exp $

Do not look for pandoc to avoid non-repro build output.

Commented on upstream at
  https://github.com/BestImageViewer/geeqie/pull/1051

--- meson.build.orig	2022-08-12 09:32:25.000000000 +0000
+++ meson.build
@@ -485,7 +485,7 @@ else
     summary({'ChangeLog' : ['ChangeLog, ChangeLog.html created:', false]}, section : 'Documentation', bool_yn : true)
 endif
 
-pandoc = find_program('pandoc', required : false)
+pandoc = find_program('pandoc-really-do-not-find-pandoc', required : false)
 if pandoc.found()
     run_command(find_program('gen_readme.sh'), meson.source_root(), meson.build_root(), check : false)
 
